packages/@tinacms/cli/src/buildTina/index.ts

```diff
@@ -98,6 +98,9 @@ export const buildSetupCmdAudit = async (
     ? new FilesystemBridge(rootPath)
     : new AuditFileSystemBridge(rootPath)
 
+  if (!fs.pathExistsSync(path.join(rootPath, '.tina', '__generated__'))) {
```

Comment:
For some reason (not sure why) we have ran into windows issues with similar code.
I have had better success with the [`ensureDir`](https://github.com/jprichardson/node-fs-extra/blob/HEAD/docs/ensureDir.md) function from fs-extra, might be worth switching to that.